src/phoenix/trace/filter.py

```diff
@@ -79,8 +79,17 @@ def _allowed_replacements() -> Iterator[Tuple[str, ast.expr]]:
         )
     }.items():
         yield source_segment, ast_replacement
-        yield "attributes." + source_segment, ast_replacement
-        yield "span.attributes." + source_segment, ast_replacement
+
+    for source_segment, ast_replacement in {
```

Comment:
refactor: This list of source_segments and ast_replacement seem largely static as it's derived off the `core.traces`. Since it's a static list - i think it will both simplify the code and make this section more readable if the list is computed once and then yielded out off of the pre-computed list

Reply:
Yea, I'm gonna refactor this as indicated in my other comment.